src/VolleyManagement.Backend/VolleyManagement.UI/Areas/WebAPI/ViewModels/Schedule/ScoreViewModel.cs

```diff
@@ -0,0 +1,48 @@
+using VolleyManagement.Domain.GamesAggregate;
+
+namespace VolleyManagement.UI.Areas.WebAPI.ViewModels.Schedule
+{
+    public class ScoreViewModel
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ScoreViewModel"/> class that contains default score.
+        /// </summary>
+        public ScoreViewModel()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ScoreViewModel"/> class that contains specified score.
+        /// </summary>
+        /// <param name="home">Score of the home team.</param>
+        /// <param name="away">Score of the away team.</param>
+        /// <param name="isTechnicalDefeat">Indicating whether the technical defeat has taken place.</param>
+        public ScoreViewModel(byte home, byte away, bool isTechnicalDefeat = false)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use the overloading mechanism instead of the optional parameters. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2360)